app/jobs/sync/bitflyer/update_missing_deposit_job.rb

```diff
@@ -0,0 +1,22 @@
+class Sync::Bitflyer::UpdateMissingDepositJob
+  include Sidekiq::Worker
+  sidekiq_options queue: :default, retry: true
+
+  def initialize(channel_id)
```

Comment:
You need to pass the channel_id to the perform method for this to work with sidekiq.

Reply:
The `@channel` object is shared between the initializer and the perform